packages/terra-search-field/src/SearchField.module.scss

```diff
@@ -1,13 +1,34 @@
 :local {
   .searchfield {
-    display: inline-flex;
+    display: flex;
+    min-width: 0;
+    width: var(--terra-search-field-max-width, 15.31em);
+
+    .clear {
+      background: transparent;
+      border-bottom-left-radius: var(--terra-search-field-clear-border-bottom-left-radius, 0);
+      border-bottom-right-radius: var(--terra-search-field-clear-border-bottom-right-radius, 0);
+      border-color: var(--terra-search-field-clear-border-color, #dedfe0);
+      border-left: var(--terra-search-field-clear-border-left, 0);
+      border-right: var(--terra-search-field-clear-border-right, 0);
```

Comment:
I'm seeing the X jump and you focus in and out of the input. This is due to changing of the border from 0 to 1. We will need to start border-right with a width of 1px to prevent the jump.

Reply:
Updated: [83eee68](https://github.com/cerner/terra-core/pull/2317/commits/83eee6882b20fa5c793d89e7eea8d429e46ebd52).